### DISCUSSION TITLE
App Downloading a Different Version of the Song

### DISCUSSION THREAD

**Discussion by axlsaludo** [Shabinder/SpotiFlyer#1687] [General]
Hello I've been using the Spotiflyer Desktop Software for about a month now and i just realized that some of the things that I've downloaded are not the song version i want to download or the song is just sometimes really different i experienced this multiple times and hoping you could get a fix some time soon.

**Comment by Arkya29** [comment]
I guess it's because Spotiflyer mostly gets songs from Youtube Music, not Spotify.

**Reply by axlsaludo** [reply]
oh that explains it, thanks
